src/needleman_wunch.jl

```diff
@@ -1,15 +1,38 @@
 # Labs 3 & 4
 
 """
-    nwscore(base1, base2)
+    nwscore(base1, base2; match, mismatch, gap) --> Int
+
+
+    Accepts two bases and possibly reassigned match, mismatch, and gap scores, then returns the correct score based on the original Needleman Wunch paper.
+    If no bases are given, an error is thrown.
+
+    Examples
+    =========
+    nwscore(base, nothing) --> -1
+    nwscore(base, nothing; gap = -2) --> -2
+    nwscore('A', 'T'; mismatch=-2, match = 3, gap=-2) --> -2
 
-Your documentation here.
 """
-function nwscore(base1::Char, base2::Char)
-    
+function nwscore(base1::Char, base2::Char; match::Int = 1, mismatch::Int = -1, gap::Int = -1)
```

Comment:
Do you want to limit your scoring system to integers? That's fairly common actually, but there's no reason in principle that you couldn't use float values as scores.